config/mainnet.json

```diff
@@ -365,7 +439,7 @@
   "delay": "40",
   "initialGasOffsetGwei": "30",
   "dynamicGasCoefficient": "1.125",
-  "maxGasLimit": "12450000",
+  "maxGasLimit": "2000000",
```

Comment:
This should limit gas burn.  LP `take()` calls appear to execute under 1.2 million gas.